### PR TITLE
Bump OptiX to 5.1.1

### PR DESCRIPTION
**Pull request by ppodhajski**

### CODE REVIEW

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
@matz-e do you have some idea why the CI does not pass? It tries to fetch this file: /gpfs/bbp.cscs.ch/project/proj3/development/deployment/NVIDIA-OptiX-SDK-5.1.1.sh
instead of: /gpfs/bbp.cscs.ch/project/proj3/development/deployment/NVIDIA-OptiX-SDK-5.1.1-linux64-25109142.s

File: var/spack/repos/builtin/packages/optix/package.py [hunk 1]
```diff
@@ -8,11 +8,11 @@
 class Optix(Package):
     """Nvidia OptiX is a ray tracing API."""
     homepage = "https://developer.nvidia.com/optix"
-    version('5.0.1','a1be032f246654ec4e4978b8c30e7774', extension='sh', expand=False)
-    url = "file:///gpfs/bbp.cscs.ch/project/proj3/development/deployment/optix.sh"
+    version('5.1.1','6e0671d49d08289c5c1ceceb9c505bbf', extension='sh', expand=False)
+    url = "file:///gpfs/bbp.cscs.ch/project/proj3/development/deployment/NVIDIA-OptiX-SDK-5.1.1-linux64-25109142.sh"
     phases = [ 'install']
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/optix/package.py:13]: I am bit surprised to see the package using file from GPFS! 

Before debugging, could we replace this? See [resource](https://spack.readthedocs.io/en/latest/packaging_guide.html#resources-expanding-extra-tarballs).

Edit : I overlooked. If this is only file, we could have small repo with that script (or other ways to handle it).

**Pull request closed without merging**